### ISSUE TITLE
No sound for notifications

### ISSUE THREAD

**Issue by borghal** [aebruno/whisperfish#40]
Just updated to v2.0, thank you!

I don't get any sound with notifications; LED and notifications in event screen are displayed, though. I'm on 2.0.1.11 and had the same issue from day one with v1.1 as well. I installed from the releases page.

Cheers!

**Comment by aebruno** [comment]
@borghal Yes this has been bugging me too. I have this fixed and will release another build shortly.

**Comment by borghal** [comment]
@aebruno Great! Thank you for the lightning fast fix!

**Comment by d9h02f** [comment]
Now it's using the SMS notification sound, I think the IM sound would be better.
